### PR TITLE
Chore 203/sonarqube integration

### PR DESCRIPTION
**Pull request by meyerinj**
#203 
SonarQube will now be run on push to a new branch. Findings can be viewed here: https://sonarqube.cloudops.objectcomputing.com/dashboard?id=oci-checkins

Developers can see these findings as a change to correct at time of push.

Alternatively this can be done on merge to develop in which case BAs would make stories to address findings when appropriate.

### CODE REVIEW

File: server/src/main/resources/application.yml [hunk 1]
```diff
@@ -70,4 +70,4 @@ flyway:
   enabled: enabled
   datasources:
     default:
-      locations: classpath:db
+      locations: classpath:db/common
```
Comment (bhatiam12) [inline server/src/main/resources/application.yml:73]: why did we change the class path ?

Comment (meyerinj) [reply server/src/main/resources/application.yml:73]: This isn't a change to the classpath; it's a change to where on the classpath we look for scripts. This is done so we can split scripts into different directories under db to control which get run under what circumstance.

Comment (meyerinj) [reply server/src/main/resources/application.yml:73]: ...But should not be part of this story. A different branch leaked onto this one.

Comment (bhatiam12) [reply server/src/main/resources/application.yml:73]: I guess that was more of my concern that why it was changed as part of this story :)

**Review by Maroju100 — APPROVED** [review]

File: .github/workflows/gradle-build-feature.yml [hunk 2]
```diff
@@ -19,5 +19,10 @@ jobs:
...
-      - name: Gradle runs tests
-        run: ./gradlew check
\ No newline at end of file
+      - name: Do SonarQube checks
+        run: ./gradlew check jacocoTestReport sonarqube -Dsonar.projectKey=$SONAR_PROJECT_KEY -Dsonar.host.url=$SONAR_URL -Dsonar.login=$TOKEN
+        env:
+          SONAR_PROJECT_KEY: oci-labs_oci-checkins
+          SONAR_ORG: oci-labs
+          SONAR_URL: https://sonarcloud.io
+          TOKEN: 76d54e72c4fbab6a5d7928cc95e2e9248986b1c5
```
Comment (bhatiam12) [inline .github/workflows/gradle-build-feature.yml:28]: should this token be stored in secrets ?

File: web-ui/src/pages/CheckinsPage.test.js [hunk 3]
```diff
@@ -1,11 +1,3 @@
...
 import CheckinsPage from "./CheckinsPage";
 import { AppContextProvider } from "../context/AppContext";
-
-it("renders correctly", () => {
-    <AppContextProvider>
-        snapshot(
-        <CheckinsPage />
-        );
-    </AppContextProvider>;
-});
```
Comment (mkimberlin) [inline web-ui/src/pages/CheckinsPage.test.js:11]: How'd this get in here?

Comment (meyerinj) [reply web-ui/src/pages/CheckinsPage.test.js:11]: Sonar was complaining about it and I was hoping to force the workflow to succeed by commenting it out. Then I forgot to undo it before I pushed.

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**